src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const value = 40;
+  const minimumDiscount = 20;
+  const maximumDiscount = 50;
+  const basePrice = days * value;
+
+  if (days <= 2) {
+    return basePrice;
+  } else if (days >= 3 && days <= 6) {
+    return basePrice - minimumDiscount;
+  } else if (days >= 7) {
+    return basePrice - maximumDiscount;
```

Comment:
The condition 'days >= 7' can be simply 'else' because you have already handled the other cases with return statements.